docker/data-science/notebooks/engine/solr/ltr.py

```diff
@@ -69,7 +48,30 @@ def upload_model(self, collection, model):
         requests.get(f"{SOLR_URL}/admin/collections?action=RELOAD&name={collection.name}&wt=xml")
         return response    
     
-    def generate_ltr_model(self, feature_store, model_name, feature_names, means, std_devs, weights):
+    def log_query(self, collection, featureset, ids, options={}, id_field="id", fields=None, log=False):
+        efi = " ".join(f'efi.{k}="{v}"' for k, v in options.items())
+        if not fields:
+            fields = [id_field]
+        fields.append(f"[features store={featureset} {efi}]")
+        request = {
+            "query": "{{!terms f={}}}{}".format(id_field, ",".join(ids)) if ids else "*:*",
+            "return_fields": fields,
+            "limit": 1000,
+            "query_parser": "default"
```

Comment:
note to trey: verify whether we can accomplish this without the query_parser param (should be able to configure edismax to do it, but may not be worth the hassle).

Reply:
I've altered this request to not require the "default" param